content/_index.md

```diff
@@ -0,0 +1,23 @@
+# Linux as Firmware
+**Reinventing the wheel by implementing drivers for firmware again and again.**</br>
+**Not with LinuxBoot™ !**
```

Comment:
could you incorporate or respond to the comments i made on the other PR?

Reply:
Done